modules/testsuite/cxf-tests/src/test/java/org/jboss/test/ws/jaxws/cxf/spring/ClientSpringAppTestCase.java

```diff
@@ -45,7 +44,8 @@
  * @author alessio.soldano@jboss.com
  * @since 02-Apr-2012
  */
-@RunWith(Arquillian.class)
+@ExtendWith(ArquillianExtension.class)
+@EnabledOnJre(JRE.JAVA_17)
```

Comment:
As SpringFramework requires JDK17 , It would be better to enable this test with java 21 too.  Can this enable with java21 ?